manifests/repo.pp

```diff
@@ -0,0 +1,26 @@
+class squid::repo
+{
+  if $::squid::manage_repo {
+    case $::osfamily {
+      'RedHat': {
+        case $::operatingsystemmajrelease {
+          '6','7': {
+            yumrepo { 'squid':
+              name     => 'Squid',
+              descr    => 'Squid repo for CentOS Linux - $basearch',
+              baseurl  => 'http://www1.ngtech.co.il/repo/centos/$releasever/$basearch/',
```

Comment:
OK.  It's the repo listed at http://wiki.squid-cache.org/SquidFaq/BinaryPackages, so I had thought it was a good choice.  I've been using it in some manual installations over the last few months.  Sometime in the last 48 hours the repo has vanished.  I might try the squid mailing list if it doesn't reappear.